xml/System.Windows.Media.Animation/IAnimation.xml

```diff
@@ -35,13 +35,14 @@
         <Parameter Name="animationClock" Type="System.Windows.Media.Animation.AnimationClock" />
       </Parameters>
       <Docs>
-        <param name="defaultOriginValue">To be added.</param>
-        <param name="defaultDestinationValue">To be added.</param>
-        <param name="animationClock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+          <param name="defaultOriginValue">The suggested origin value to provide the animation, which is used if the animation does not have its own concept of a start value. If this animation is the first in a composition chain, it is the snapshot value, if one is available, or the base property value, if it is not; otherwise, it is the value returned by the previous animation in the chain with an animation clock that is not <see cref="F:System.Windows.Media.Animation.ClockState.Stopped" />.</param>
+          <param name="defaultDestinationValue">The suggested destination value to provide the animation, which is used if the animation does not have its own concept of an end value. It is the base value if the animation is in the first composition layer of animations on a property; otherwise, it is the output value from the previous composition layer of animations for the property.</param>
```

Comment:
```suggestion
        <param name="defaultDestinationValue">The suggested destination value to provide the animation, which is used if the animation does not have its own concept of an end value. It is the base value if the animation is in the first composition layer of animations on a property; otherwise, it is the output value from the previous composition layer of animations for the property.</param>
```